app/payment/session.go

```diff
@@ -114,6 +123,24 @@ func RespondPayChProposal(pctx context.Context, s perun.SessionAPI, proposalID s
 	return s.RespondChProposal(pctx, proposalID, accept)
 }
 
+// SubPayChCloses registers a subscription for payment channel closes.
```

Comment:
```suggestion
// SubPayChCloses subscribe for payment channel closes.
```

Reply:
Changed to sets up a subscription for ...